ergo-wallet/src/test/scala/org/ergoplatform/wallet/transactions/TransactionBuilderSpec.scala

```diff
@@ -0,0 +1,73 @@
+package org.ergoplatform.wallet.transactions
+
+import sigmastate.Values
+import sigmastate.Values.SigmaPropValue
+import sigmastate.eval._
+import sigmastate.eval.Extensions._
+import org.ergoplatform.{ErgoBox, ErgoBoxCandidate}
+
+import org.scalatest.{Matchers, PropSpec}
+import org.ergoplatform.ErgoBox.TokenId
+import scala.util.Success
+import scorex.crypto.hash.Digest32
+import org.ergoplatform.P2PKAddress
+import org.ergoplatform.wallet.mnemonic.Mnemonic
+import org.ergoplatform.wallet.secrets.ExtendedSecretKey
+import org.ergoplatform.ErgoAddressEncoder
+
+class TransactionBuilderSpec extends PropSpec with Matchers {
+  import TransactionBuilder.buildUnsignedTx
+
+  implicit val addressEncoder = new ErgoAddressEncoder(
+    ErgoAddressEncoder.TestnetNetworkPrefix
+  )
+
+  val seedStr                       = "edge talent poet tortoise trumpet dose"
+  val seed: Array[Byte]             = Mnemonic.toSeed(seedStr)
+  val rootSecret: ExtendedSecretKey = ExtendedSecretKey.deriveMasterKey(seed)
+
+  val currentHeight    = 0
+  val minFee           = 1000000L
+  val minChangeValue   = 1000000L
+  val minerRewardDelay = 720
+
+  val TrueLeaf: SigmaPropValue = Values.TrueLeaf.toSigmaProp
```

Comment:
When it is SigmaProp, why not call it TrueProp?

Reply:
Done.